cfme/utils/version.py

```diff
@@ -129,13 +129,13 @@ def dependent(default_function):
     return m
 
 
-def pick(v_dict, active_version=None):
+def pick(v_dict, active_version=None, appliance_version=None):
     """
     Collapses an ambiguous series of objects bound to specific versions
     by interrogating the CFME Version and returning the correct item.
     """
     # convert keys to Versions
-    active_version = active_version or current_version()
+    active_version = appliance_version or active_version or current_version()
```

Comment:
**REQUIRED** remove the `appliance_version` argument as it has the same usage as the active_version argument